models/CESM/doc/setup_guidelines.rst

```diff
@@ -0,0 +1,242 @@
+    CESM. We recommend a tiny ensemble to start with, to more quickly test whether everything is in order.
+9.  After convincing yourself that the CESM+DART framework is working with no_assimilate.csh, activate the assimilation
+    by changing CESM's env_run.xml:DATA_ASSIMILATION_SCRIPT to use assimilate.csh.
+10. After the first forecast+assimilation cycle finishes correctly, change the input.nml, env_run.xml and env_batch.xml
+    to do additional cycle(s) without the perturbation of the initial state, and with using the just created restart
+    files. You may also want to turn on the st_archive program. Instructions are in setup_hybrid and
+    cam-fv/work/input.nml.
+11. Finally, build a new case with the full ensemble, activate the assimilate.csh script and repeat the steps in step
+    10.
+
+Output directory
+----------------
+
+CESM's short term archiver (st_archive) is controlled by its ``env_archive.xml``. DART's setup scripts modify that file
+to archive DART output along with CESM's.
+DART's output is archived in ``$arch_dir/dart/{hist,rest,logs,...}``, where arch_dir is defined in
+``setup_{hybrid,advanced}``, ``hist`` contains all of the state space and observation space output, and ``rest``
+contains the inflation restart files.
+
++---------------------------------------+---------------------------------------+---------------------------------------+
+| ::                                    | User                                  | Location of scripts and pass-through  |
+|                                       |                                       | point for files during execution.     |
+|    Central directory                  |                                       | Typically named according defining    |
+|                                       |                                       | characteristics of a \*set\* of       |
+|                                       |                                       | experiments; resolution, model, obs   |
+|                                       |                                       | being assimilated, unique model state |
+|                                       |                                       | variables, etc.                       |
++---------------------------------------+---------------------------------------+---------------------------------------+
+
+| -->
+
+The cam-XX assimilate.csh scripts also make a copy of the obs_seq.final files in a scratch space
+($scratch/$case/Obs_seqs) which won't be removed by CESM's long term archiver, if that is run.
+
+Helpful hints
+-------------
+
+Space requirements
+------------------
+
+Space requirements (Gb per ensemble member) for several CAM resolutions.
+
+| 
+
+There are, no doubt, things missing from these lists, so don't struggle too long before contacting dart'at'ucar.edu.
+
+Useful terms found in this web page.
```

Comment:
This list is absent from the .rst version ... it exists as a list in the Manhattan (raw html) version.

Reply:
@kdraeder  - can you handle this (later?)